### ISSUE TITLE
Should discounts be hidden from some customers?

### ISSUE THREAD

**Issue by lucasjkr** [lucasjkr/opencommerce#51]
opencart/opencart#6131

**Comment by arnisjuraga** [comment]
This seems to be kind of huge bug? :)
I mean - if you "don't" show prices for customers (option set in settings),  that will still display discounts.

**Comment by lucasjkr** [comment]
Didn't look at it yet, was just perusing OC and Copona issues... Making notes of what to come back to!

**Comment by tom-it** [comment]
yes, this is a bug, if you don't want to show prices for non registered users, you shouldn't be showing discounted prices either. the pull request should fix it.
